### ISSUE TITLE
HTTP Basic Auth Prompt problems

### ISSUE THREAD

**Issue by gardar** [varjolintu/keepassxc-browser#52]
## Current Behavior
Websites with Basic Auth prompts seem to hang when KeePassXC-Browser is enabled and connected to KeePassXC.
The prompt does not get displayed and the website just "hangs", nothing gets loaded.

If the KeePassXC database is closed, then the website loads successfully.

The status bar shows:
"Waiting for extension KeePassXC-Browser"

I am experiencing this both on Version 64.0.3282.119 (Developer Build) (64-bit) and Firefox 58.0.1 (64-bit) on ArchLinux with KeePassXC - Version 2.2.4 - Revision: bed921c

## General Info
KeePassXC - Version 2.2.4 - Revision: bed921c
keepassxc-browser - 0.5.1

Operating system: ArchLinux
Browser: Chromium Version 64.0.3282.119 (Developer Build) (64-bit) and Firefox 58.0.1 (64-bit) 
Proxy used: keepassxc-proxy yes - other proxies no

**Comment by varjolintu** [comment]
This certainly needs some tuning. I will check this out and see what I can do.

**Comment by varjolintu** [comment]
Does anything happen if you click the extension icon?

**Comment by gardar** [comment]
Nothing happens when I click the extension icon, both in firefox and chromium.

Strangely this morning I did get a http auth prompt in chromium when I opened a webpage that has http auth, yesterday that didn't happen and it did just hang forever.
Firefox still hangs when I try to access the same page.

Both Firefox and Chromium also just hang if I click a link/buttun that points to a http auth page or if I access a webpage that redirects me automatically to a http auth page.
